src/components/app/data/utils.js

```diff
@@ -558,6 +558,11 @@ export function isArchived(courseRun) {
   return false;
 }
 
+// These are the standard rules used for determining whether a run is "available".
+export const standardAvailableCourseRunsFilter = (courseRun) => (
```

Comment:
It looks like this function was abstracted to a standalone export to be used directly within test assertions. I might recommend not using this function within the tests to ensure adequate coverage / regression testing on its logic.